neurom/analysis/morphtree.py

```diff
@@ -264,6 +264,40 @@ def trunk_length(tree):
         return 0.0
 
 
+def trunk_vector(tree, soma):
+    '''Vector of trunk defined as (initial tree point - soma center)
+       of the tree.
+    '''
+    return mm.vector(tree.value, soma.center)
+
+
+def trunk_elevation(tree, soma):
```

Comment:
Better keep "trunk_elevation", "trunk_azimuth" and "trunk_direction" together and not mix with "initial"! :-)

So, for my education: "trunk" means "the first cross section", and not "the first segment".